packages/driver/cypress/integration/e2e/multi-domain/navigation_spec.ts

```diff
@@ -190,6 +190,7 @@ describe('navigation events', () => {
     })
 
     // TODO: this test should re revisited with the cypress in cypress tests available in 10.0
```

Comment:
The multi-origin stuff doesn't really work with the isolated runner tests in 9.x due to there being another version of the driver in the spec bridge that's not currently taken into account in the isolated runner. It's not really worth adding the support for it in the isolated runner since it's being replaced in 10.0.